File: muridku/muridku.backend/GlobalDbProcessor/Muridku.QueryRequestReceiver/Models/Dbs/Institution.cs
namespace Muridku.QueryRequestReceiver.Models.Dbs
{
  public class Institution : ModelDbBase
  {
    public string code { get; set; }
    public string name { get; set; }
    public string type { get; set; }
    public string address { get; set; }
  }
}

File: muridku.admin/muridku/app/Models/Institution.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Institution extends Model
{
    use HasFactory;

    protected $table = 'institution';
    public $timestamps = false;

    protected $fillable = [
        'code',
        'name',
        'type',
        'address'
    ];

    public function getTypeName()
    {
        $typename = [
            'SHS' => 'SMA',
            'COL'  => 'Kampus',
            'WORK' => 'Kerja',
        ];

        return $typename[$this->type] ?? 'Tipe Tidak Diketahui';
    }
}
